### DISCUSSION TITLE
Results archive command in ORFS

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD-flow-scripts#404] [Q&A] [answered]
Hi @vvbandeira,
While using ORFS flow either flow completes or fails for particular config.mk.

Whether is there any command,  I can use to archive my all logs/reports/results that specific runs?

In openlane there is command `-tag ` switch for results to save for different run.

In ORFS is there any such switch. Because by default i see `base` directory created every time. I need to `make clean_all` to re-run same design with different configuration.

Thanks,
Vijayan K

**Comment by vvbandeira** [comment, marked as answer]
There are two features you can use to create a workflow that fits your needs.

First, you can create archives (tar files) with `make <SCRIPT_NAME>_issue` where `<SCRIPT_NAME>` is the name of the file in `flow/scripts`, e.g., `make cts_issue`. This will create a tar file with logs, results and the necessary files to reproduce. This is more suitable to create testcases to report issues to developers.

Second, you can use the `FLOW_VARIANT` variable on your `config.mk` or directly on the command line `make FLOW_VARIANT=<TAG> [...]`. This will create a folder with the same name to store files, e.g., `logs/sky130hd/gcd/<TAG>/*.log`.
